config/singleFrameDriver.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+from lsst.utils import getPackageDir
+
+config.processCcd.load(os.path.join(getPackageDir("obs_decam"), config, processCcd.py))
```

Comment:
config -> "config"
processCcd.py  -> "processCcd.py" 
(They need to be in quotation marks)